docs/_tutorials/java-client.md

```diff
@@ -23,39 +23,21 @@ The examples below make a few assumptions for simplicity:
-
-1.	Open the online Swagger editor: http://editor.swagger.io/#/
-2.	In the editor’s **File** menu, select **Import File ...**
-3.	Using the pop-up dialog, **Browse...** to the location of the `swagger.json` and **Open** the file
-4.	In the editor’s **Generate Client** menu, select the desired language from the list provided
-5.	A generated client library will start to download.
+The [Generating the SEMP Client Library tutorial]({{ site.baseurl }}/generate-semp-client-lib/) provides details about generating the latest version of the library. The Gradle build script at the end of this tutorial will automatically generate the required SEMP Java Client Library for the first time if it doesn't exist yet, as part of the build.
 
 ## Client Library Basics
 
 Before jumping to specific tasks like creating a Client Username, we will first introduce a few basic concepts common to the SEMP client library. If you have not already, you can learn more about these concepts and SEMP in general by checking out the [SEMP Concepts]({{ site.docs-architecture }}){:target="_top"}.
 
 ### Initializing the SEMP API
 
-Before sending any commands, you must initialize the client library. There is a close relationship between the tags used in the SEMP specification and the SEMP client library classes offered. Each tag, like `msgVpn`, is compiled into a class in the `io.swagger.client.api` package. Because of this, you have the option to work with the full SEMP API or subsets as controlled by the tags in the SEMP specification. You should determine which tag best fits your use case and use this client API class. For this introduction, we will use the `msgVpn` tag and the `MsgVpnApi` class because configuring a Solace message router message-VPN is a very common task in a dev-ops workflow.
+Before sending any commands, you must initialize the client library. There is a close relationship between the tags used in the SEMP specification and the SEMP client library classes offered. Each tag, like `msgVpn`, is compiled into a class in the `com.solace.sempclient.samplelib.api` package. Because of this, you have the option to work with the full SEMP API or subsets as controlled by the tags in the SEMP specification. You should determine which tag best fits your use case and use this client API class. For this introduction, we will use the `msgVpn` tag and the `MsgVpnApi` class because configuring a Solace message router message-VPN is a very common task in a dev-ops workflow.
 
 Before you can send any commands to the Solace message router, you need an instance of the SEMP API. The following code shows you how to create such an instance and set the SEMP username and password. This will connect over HTTP.
 
 ```
-import io.swagger.client.ApiClient;
-import io.swagger.client.api.MsgVpnApi;
+import com.solace.sempclient.samplelib.ApiClient;
```

Comment:
Needs to be com.solace.labs....

This isn't a supported Solace product, so it shouldn't generate libraries in the com.solace space but instead use the labs.